whatsagent/api/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
```

Comment:
We're not going to use django's admin panel, remove this file.

Reply:
Removed